src/gml/GML.jl

```diff
@@ -33,7 +26,7 @@ function mk_parser(string_input)
         comment = P"(#.*)?"
 
         # we need string input as we match multiple lines
-        if string_input && ParserCombinator.FAST_REGEX
```

Comment:
@CarloLucibello  Do you know why `ParserCombinator.FAST_REGEX` was removed from this line?

Reply:
Ah becaue it is always true now